### PR TITLE
feat: Instance Settings API

### PR DESCRIPTION
**Pull request by gkats**
API operations for instance settings. Update instance settings, update instance restrictions and update instance organization settings.

### CODE REVIEW

File: clerk_test.go [hunk 1]
```diff
@@ -279,6 +279,35 @@ func TestBackendCall_SuccessfulResponse_GetRequest(t *testing.T) {
 	assert.JSONEq(t, rawJSON, string(resource.Response.RawJSON))
 }
 
+// TestBackendCall_SuccessfulResponse_EmptyResponse tests successful
+// responses with no response body.
+func TestBackendCall_SuccessfulResponse_EmptyResponse(t *testing.T) {
```
Comment (georgepsarakis) [inline clerk_test.go:284]: ➕

File: cmd/gen/main.go [hunk 2]
```diff
@@ -129,8 +129,8 @@ func main() {
 const lineStartsWith = "func (c *Client) "
 
 var nameRE = regexp.MustCompile("^\\w+\\(")
-var argsRE = regexp.MustCompile("^.+\\)\\s\\(")
-var returnRE = regexp.MustCompile("^.+\\)")
+var argsRE = regexp.MustCompile("^.+\\)\\s[\\(e]")
+var returnRE = regexp.MustCompile("^.+\\s{")
```
Comment (georgepsarakis) [inline cmd/gen/main.go:133]: 💡 Mostly an idea/suggestion for possible future improvement, definitely not a current action item: we can use AST parsing to perform code generation in a possibly more maintainable way in the future: https://eli.thegreenplace.net/2021/rewriting-go-source-code-with-ast-tooling/

**Review by georgepsarakis — APPROVED** [review]

File: cmd/gen/main.go (hunk 2 above)
Comment (gkats) [reply cmd/gen/main.go:133]: Yes, this "generator" is a disgrace. :disappointed:

Comment (georgepsarakis) [reply cmd/gen/main.go:133]: Gets the work done is what matters 🥇

**Pull request merged**